src/databricks/labs/ucx/install.py

```diff
@@ -486,6 +524,17 @@ def _job_settings(self, step_name: str, dbfs_path: str):
             "tasks": [self._job_task(task, dbfs_path) for task in tasks],
         }
 
+    @staticmethod
+    def _apply_cluster_overrides(settings: dict[str, any], overrides: dict[str, str]) -> dict:
```

Comment:
did you change it? why does it appear in the diff?

Reply:
I did.